runtime.txt

```diff
@@ -1 +1 @@
-python-3.4.3
+python-3.7.2
```

Comment:
Last time I tried to upgrade to 3.5 or 3.6 I couldn't get it to work

Reply:
I didn't spend too much time on it though

Reply:
https://github.com/sqlalchemy-redshift/bigcrunch/issues/1
Yeah so for one, it will need aiobotocore, and aiohttp/starlette to support >py3.5 as the yieldfrom project is dead